src/main/java/com/google/sps/servlets/NewMemberServlet.java

```diff
@@ -66,6 +69,12 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
         datastore.put(familyEntity);
 
+        String calendarID = (String) familyEntity.getProperty("calendarID");
```

Comment:
`calendarID` is used across multiple servlets. It might be worth creating a common `Constants.class` where you define such strings and use that across all your servlets.

Reply:
This seems like a big refactoring change so I will do this in a different PR